checker/src/org/checkerframework/checker/lock/qual/PolyGuardedBy.java

```diff
@@ -0,0 +1,36 @@
+/*
+TODO: Implement the functionality for @PolyGuardedBy and uncomment this.
```

Comment:
Thanks for the explanation.  One key difference between `@GuardSatisifed` and `@PolyGuardedBy` is that when `@PolyGaurdedBy` is the LHS of an assignment, the locks guarding the RHS do not have to be held.  Would you please add this to the Javadoc?